### PR TITLE
node modules weren't transpiled by babel

### PR DESCRIPTION
**Pull request by mstoykov**
This was a complex issue that required more or less the changes to
webpack and the move of the babel config to fix.

babel.config.json changes are there as on a particular input some
plugins were introducing errors.

### CODE REVIEW

File: babel.config.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    ],
+    "plugins": [
+        "@babel/plugin-transform-exponentiation-operator",
+        "@babel/plugin-transform-regenerator",
+        "@babel/plugin-transform-classes",
+        "@babel/plugin-transform-object-super",
+        "@babel/plugin-transform-async-to-generator",
+        "@babel/proposal-class-properties",
+        "@babel/proposal-object-rest-spread",
+        [ "@babel/plugin-proposal-unicode-property-regex",       { useUnicodeFlag: false }]
```
Comment (mstoykov) [inline babel.config.json:13]: This one in particular works around https://github.com/dlclark/regexp2/pull/52 no idea if it breaks something else though

File: babel.config.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "@babel/typescript"
+    ],
+    "plugins": [
+        "@babel/plugin-transform-exponentiation-operator",
+        "@babel/plugin-transform-regenerator",
+        "@babel/plugin-transform-classes",
+        "@babel/plugin-transform-object-super",
+        "@babel/plugin-transform-async-to-generator",
+        "@babel/proposal-class-properties",
+        "@babel/proposal-object-rest-spread",
```
Comment (mstoykov) [inline babel.config.json:12]: I think we can also drop all of those now that goja has support for practically all of them :thinking:

**Comment by ppcano** [comment]
@mstoykov I plan to look at it on this quarter.

Also, review https://github.com/grafana/k6-template-typescript/pull/34 and document the supported ES* features.

**Comment by ppcano** [comment]
I have recently updated the project dependencies. 

Currently, this project does not include an example resolving `node_modules`. If you are looking for that, refer to the [k6-rollup-example](https://github.com/grafana/k6-rollup-example).

I am closing this PR due to inactivity. Please, reopen if necessary.

**Pull request closed without merging**